jigasi/rootfs/etc/s6-overlay/scripts/autoscaler-sidecar

```diff
@@ -0,0 +1,10 @@
+#!/command/with-contenv bash
+
+if [[ -n "$AUTOSCALER_URL" ]] && [[ -f "/run/autoscaler-sidecar/config" ]]; then
+    DAEMON="/usr/bin/node /usr/share/jitsi-autoscaler-sidecar/app.js"
+    exec /bin/bash -c ". /run/autoscaler-sidecar/config && exec $DAEMON"
```

Comment:
I think this can be simplified with something like: `env $(cat /run/autoscaler-sidecar/config| xargs) $DAEMON`

Reply:
Maybe as an addtional PR.
What do you think?